managed_vms/django_tutorial/README.md

```diff
@@ -0,0 +1,103 @@
+# Getting started with Django on Google Cloud Platform
+
+This repository is an example of how to run a [Django](https://www.djangoproject.com/) 
+app on Google Managed VMs. It uses the [Writing your first Django app](https://docs.djangoproject.com/en/1.9/intro/tutorial01/) as the example app to deploy.
+
+
+## Setup the database
+
+This tutorial assumes you are setting up Django using a SQL database, which is the easiest way to run Django. If you have an existing SQL database running, you can use that, but if not, these are the instructions for creating a managed MySQL instance using CloudSQL.
+
+
+* Create a [CloudSQL instance](https://console.cloud.google.com/project/_/sql/create)
+
+* In the instances list, click your Cloud SQL instance.
+
+* Click Access Control.
+
+* In the IP address subsection, click Request an IPv4 address to enable access to the Cloud SQL instance through an 
+IPv4 address. It will take a moment to initialize the new IP address.
+
+* Also under Access Control, in the Authorization subsection, under Allowed Networks, click the add (+) button .
+
+* In the Networks field, enter 0.0.0.0/0. This value allows access by all IP addresses.
+
+* Click Save.
+
+* Click New user.
+
+* Enter a username and password for the application. For example, name the user "pythonapp" and give it a randomly 
+generated password.
+
+* Click Add.
+
+* Click Databases and then click New database.
+
+* For Name, enter the name of your database (for example, "polls"), and click Add.
+
+Once you have a SQL host, configuring mysite/settings.py to point to your database. Change `your-database-name`, 
+`your-database-user, `your-database-host`, and`your-database-password` to match the settings created above. Note the 
+instance name is not used in this configuration, and the host name is the IP address you created.
+
+## Running locally
+
+First make sure you have Django installed. It's recommended you do so in a 
+[virtualenv](https://virtualenv.pypa.io/en/latest/). The requirements.txt
+contains just the Django dependency.
+
+`pip install -r requirements.txt`
+
+Once the database is setup, run the migrations.
+
+`python manage.py migrate`
+
+If you'd like to use the admin console, create a superuser.
+
+`python manage.py createsuperuser`
+
+The app can be run locally the same way as any other Django app. 
+
+`python manage.py runserver`
+
+Now you can view the admin panel of your local site at
+
+`http://localhost:8080/admin`
+
+## Deploying
+
+Since the Django development server is not built for production, our container uses the Gunicorn server. Since Gunicorn doesn't serve static content,
+the static content is instead served from Google Cloud Storage.
```

Comment:
Slight overkill, but the alternative is that our sample was using the development server. Maybe once bookshelf gets done we can think about changing this back, but for now with just 1 or 2 samples I think showing how to setup gunicorn is worth it.